bitcoin/CHANGELOG.md

```diff
@@ -57,6 +57,8 @@ This release is big, to help users upgrade we wrote a blog post, please see http
      contains a `LockTime` but is `Ord`, we have manually sorted the locktimes based on
      their consensus encoding. This ordering is somewhat arbitrary -- there is no total
      ordering on locktimes since they may be measured in either blocks or seconds.
+  - [Remove TxOut Default implementation](https://github.com/rust-bitcoin/pull/1838). The behavior
```

Comment:
This is wrong place to put this - it's the old version.

We do changlogs during release, so feel free to just remove this.

Reply:
Oops my mistake, I didn't know how the changelog is managed.  I can remove it.  One benefit to making the changelog part of the commit is it makes the git diff more informative for the commit.

Reply:
Yeah, please remove it. We would like to write changelog notes inside commit messages and then generate the changelog from them but that's a different topic.

Reply:
Lol, oops, I dunno how I missed this.

Reply:
done.  Here's the text for the future:

```
  - [Remove TxOut Default implementation](https://github.com/rust-bitcoin/pull/1838). The behavior
    that was used in the default trait has been moved to TxOut::NULL.
```